docs/ja/docs/tutorial/extra-data-types.md

```diff
@@ -36,7 +36,7 @@
 * `datetime.timedelta`:
     * Pythonの`datetime.timedelta`です。
     * リクエストとレスポンスでは合計秒数の`float`で表現されます。
-    * Pydanticでは「ISO 8601 time diff encoding」として表現することも可能です。<a href="https://pydantic-docs.helpmanual.io/#json-serialisation" class="external-link" target="_blank">詳細はドキュメントを参照してください</a>。
+    * Pydanticでは「ISO 8601 time diff encoding」として表現することも可能です。<a href="https://docs.pydantic.dev/latest/#json-serialisation" class="external-link" target="_blank">詳細はドキュメントを参照してください</a>。
```

Comment:
Here it could be better this one
https://docs.pydantic.dev/latest/concepts/serialization/